src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/Contracts/DeploymentOptions.cs

```diff
@@ -225,7 +225,6 @@ public DeploymentOptions()
             options.DropRoleMembersNotInSource = true;
             options.IgnoreKeywordCasing = false;
             options.IgnoreSemicolonBetweenStatements = false;
-            options.IgnoreWhitespace = false;
```

Comment:
Wouldn't this be a problem with the comment that's mentioned in the function:  "Adding these defaults to ensure behavior similarity with other tools. Dacfx and SSMS import/export wizards use these defaults."? If not DacFx, I feel it might be a problem with SSMS (based on the mentioned comment, didn't test it).

Reply:
This was originally done because there are different defaults for the different tools and we wanted to move towards having one set of defaults. However, we didn't make any other effort to have a common set of default values. In this case, I think it's better to match the default of SSDT for this option so that the comparison will have the same behavior.

Reply:
Sounds good. Thanks!